src/GraphQL/DI/ServiceLifetime.cs

```diff
@@ -0,0 +1,26 @@
+namespace GraphQL.DI
+{
+    /// <summary>
+    /// Specifies the lifetime of a service.
+    /// </summary>
+    public enum ServiceLifetime
```

Comment:
OK. https://github.com/EasyNetQ/EasyNetQ/blob/develop/Source/EasyNetQ/DI/Lifetime.cs

Reply:
Most likely I copied the xml comments from MS. Can’t remember